JS/see_article.js

```diff
@@ -73,29 +81,41 @@ function create_see_article(article) {
     let infoDiv = document.createElement('div');
     infoDiv.id = 'productDetails';
     let detailsDiv = document.createElement('div');
+    detailsDiv.id = "detailDivId";
     let buttonDiv = document.createElement('div');
     buttonDiv.id = 'button';
     let box_selectdiv = document.createElement('div');
     box_selectdiv.className = "number-input";
     let emptydiv = document.createElement('div');
     emptydiv.style = "height: 12px; ";
-
+    emptydiv.id = "emptydiv";
+    
     let image = document.createElement('img');
     image.src = article.preview;
 
     let name = document.createElement('h1');
+    name.id = "nameId";
     let nameContent = document.createTextNode(article.name);
     let price = document.createElement('h3');
-    let priceContent = document.createTextNode("Prix unitaire : " + (parseFloat(article.price)).toFixed(2) + "€");
+    price.id = 'price';
+//    let priceContent = document.createTextNode("Prix unitaire : " + (parseFloat(article.price)).toFixed(2) + "€");
+    let priceContent = document.createTextNode((parseFloat(article.price)).toFixed(2) + "€");
```

Comment:
is the use of `parseFloat().toFixed()` necessary here ?

Reply:
It's because I wanted to show the price with a pre-defined style like : 99.00€ or 45.99€. 
in the case where in the database we have an article at 99 euro, the user may write 99 and not 99.00. But if this is too superfluous, I remove it

Reply:
what is the type of article.price ? 
I agree that a having a uniform display is worth a little more computing

Reply:
String